scannergradle.properties

```diff
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-gradle/
 issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/SONARGRADL/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-gradle
-archivedVersions=2.8,3.0,3.1,3.1.1,3.2,3.3,3.4.0.2513,3.5.0.2730,4.0.0.2929,4.2.1.3168,4.3.1.3277
-publicVersions=4.4.1.3373
+archivedVersions=2.8,3.0,3.1,3.1.1,3.2,3.3,3.4.0.2513,3.5.0.2730,4.0.0.2929,4.2.1.3168,4.3.1.3277,4.4.1.3373
+publicVersions=5.0.0.4638
+
+5.0.0.4638.description=Decouple sonar task from Gradle compile tasks
+5.0.0.4638.date=2024-03-26
+5.0.0.4638.changelogUrl=https://sonarsource.atlassian.net/projects/SONARGRADL/versions/14301/tab/release-report-all-issues
+5.0.0.4638.downloadUrl=https://plugins.gradle.org/plugin/org.sonarqube/5.0.0.4638
```

Comment:
@dorian-burihabwa-sonarsource both URL formats seem to be slightly different. Perhaps not a big deal.

Reply:
Fixed